src/Compilers/VisualBasic/Portable/Symbols/SubstitutedNamedType.vb

```diff
@@ -589,6 +590,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Next
         End Function
 
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Dim originalRequiredMembers = OriginalDefinition.AllRequiredMembers
+                If originalRequiredMembers.IsEmpty OrElse originalRequiredMembers = s_requiredMembersErrorSentinel Then
+                    Return originalRequiredMembers
```

Comment:
>Return originalRequiredMembers [](http://example.com/codeflow?start=20&length=30)

Do we test this code path for both conditions that can get us here?

Reply:
This will be one unified codepath for all symbols in the next revision, more similar to C#.